src/cets_long.erl

```diff
@@ -48,28 +48,30 @@ run_spawn(Info, F) ->
 run_tracked(Info, Fun) ->
     Parent = self(),
     Start = erlang:system_time(millisecond),
-    ?LOG_INFO(Info#{what => long_task_started}),
+    ?LOG_INFO(Info#{what => task_started}),
     Pid = spawn_mon(Info, Parent, Start),
     try
         Fun()
     catch
-        Class:Reason:Stacktrace ->
+        Class:Reason:Stacktrace when element(1, Reason) =/= task_failed ->
```

Comment:
Interesting, so this guard can fail but the failure is just treated as non-matching.

Reply:
true, fixed.
So, I also changed format of assertException in some tests, after that.